relay-server/src/metric_stats.rs

```diff
@@ -107,7 +107,8 @@ impl MetricStats {
     }
 
     fn is_enabled(&self, scoping: Scoping) -> bool {
-        self.config.processing_enabled() && self.is_rolled_out(scoping.organization_id)
+        (self.config.processing_enabled() || self.config.metric_stats_enabled())
```

Comment:
nit: Should `metric_stats_enabled()` check `self.config.processing_enabled()` (that's where the behavior is documented)?

Reply:
Good idea, will update the code and adjust docs!